<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportPriKeyDlg</class>
 <widget class="QDialog" name="ImportPriKeyDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import private key</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="mTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Session</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Slot status</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="mSlotIDText"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Path</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Slot ID</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Slot list</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="6">
        <widget class="QComboBox" name="mSlotsCombo"/>
       </item>
       <item row="3" column="5">
        <widget class="QLineEdit" name="mLoginText"/>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="mSessionText"/>
       </item>
       <item row="4" column="1" colspan="4">
        <widget class="QLineEdit" name="mPathText"/>
       </item>
       <item row="4" column="5">
        <widget class="QPushButton" name="mFindBtn">
         <property name="text">
          <string>Find</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QCheckBox" name="mPubImportCheck">
         <property name="text">
          <string>Public Key Import</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Private Key</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="0">
        <widget class="QCheckBox" name="mPriPrivateCheck">
         <property name="text">
          <string notr="true">Private</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="mPriPrivateCombo"/>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="mPriUnwrapCheck">
         <property name="text">
          <string notr="true">Unwrap</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QCheckBox" name="mPriSignCheck">
         <property name="text">
          <string notr="true">Sign</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Subject</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QComboBox" name="mPriDecryptCombo"/>
       </item>
       <item row="7" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Label</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="mPriDecryptCheck">
         <property name="text">
          <string notr="true">Decrypt</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Attributes</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="mPriDeriveCheck">
         <property name="text">
          <string notr="true">Derive</string>
         </property>
        </widget>
       </item>
       <item row="4" column="5">
        <widget class="QComboBox" name="mPriSignCombo"/>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="mPriUnwrapCombo"/>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="mPriDeriveCombo"/>
       </item>
       <item row="5" column="3">
        <widget class="QComboBox" name="mPriModifiableCombo"/>
       </item>
       <item row="6" column="3">
        <widget class="QComboBox" name="mPriExtractableCombo"/>
       </item>
       <item row="5" column="5">
        <widget class="QComboBox" name="mPriSensitiveCombo"/>
       </item>
       <item row="6" column="5">
        <widget class="QComboBox" name="mPriTokenCombo"/>
       </item>
       <item row="5" column="2">
        <widget class="QCheckBox" name="mPriModifiableCheck">
         <property name="text">
          <string notr="true">Modifiable</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QCheckBox" name="mPriExtractableCheck">
         <property name="text">
          <string notr="true">Extractable</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QCheckBox" name="mPriSensitiveCheck">
         <property name="text">
          <string notr="true">Sensitive</string>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="QCheckBox" name="mPriTokenCheck">
         <property name="text">
          <string notr="true">Token</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="5">
        <widget class="QLineEdit" name="mPriLabelText"/>
       </item>
       <item row="1" column="1" colspan="5">
        <widget class="QLineEdit" name="mPriSubjectText"/>
       </item>
       <item row="2" column="1" colspan="5">
        <widget class="QLineEdit" name="mPriIDText"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Public Key</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Label</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="mPubVerifyCheck">
         <property name="text">
          <string notr="true">Verify</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="mPubTokenCheck">
         <property name="text">
          <string notr="true">Token</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QComboBox" name="mPubEncryptCombo"/>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="mPubPrivateCheck">
         <property name="text">
          <string notr="true">Private</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="mPubPrivateCombo"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Attributes</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="4">
        <widget class="QCheckBox" name="mPubWrapCheck">
         <property name="text">
          <string notr="true">Wrap</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QCheckBox" name="mPubEncryptCheck">
         <property name="text">
          <string notr="true">Encrypt</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QComboBox" name="mPubWrapCombo"/>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="mPubVerifyCombo"/>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="mPubTokenCombo"/>
       </item>
       <item row="4" column="3">
        <widget class="QComboBox" name="mPubDeriveCombo"/>
       </item>
       <item row="4" column="5">
        <widget class="QComboBox" name="mPubModifiableCombo"/>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="mPubDeriveCheck">
         <property name="text">
          <string notr="true">Derive</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QCheckBox" name="mPubModifiableCheck">
         <property name="text">
          <string notr="true">Modifiable</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="5">
        <widget class="QLineEdit" name="mPubLabelText"/>
       </item>
       <item row="1" column="1" colspan="5">
        <widget class="QLineEdit" name="mPubIDText"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImportPriKeyDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImportPriKeyDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
